src/routes/wallet/tx-template/tx-template.routes.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { Router } = require('express');
+const { query } = require('express-validator');
+const { TransactionTemplate } = require('@hathor/wallet-lib');
+const {
+  buildTemplate,
+  runTemplate,
+} = require('../../../controllers/wallet/tx-template/tx-template.controller');
+const { patchExpressRouter } = require('../../../patch');
+const { validateZodSchema } = require('../../../helpers/validations.helper');
+
+const txTemplateRouter = patchExpressRouter(Router({ mergeParams: true }));
+
+/**
+ * GET request to fetch which of the tx inputs are from the wallet.
+ * For the docs, see api-docs.js
+ */
+txTemplateRouter.post(
+  '/run',
+  validateZodSchema(TransactionTemplate),
+  query('debug').default(false).isBoolean().toBoolean(),
+  runTemplate,
+);
+
+txTemplateRouter.post(
+  '/build',
+  validateZodSchema(TransactionTemplate),
+  query('debug').default(false).isBoolean().toBoolean(),
+  query('sign').default(false).isBoolean().toBoolean(),
```

Comment:
Why use query parameter and not body, since it's a POST?

Reply:
Like the debug, query seems more reasonable to add a flag since we do not have other arguments